From 6c58c56f21995a134563cefe1e63bf81195a2786 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 2 Aug 2026 14:46:18 +0200
Subject: [PATCH] dt-bindings: media: sun6i-a31-isp: Add the A83T compatible

The A83T has the same ISP, at the same address, with the same clocks and
reset. It has a single CSI, so port@0 is required just like it is on the
V3s.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 .../devicetree/bindings/media/allwinner,sun6i-a31-isp.yaml      | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/allwinner,sun6i-a31-isp.yaml b/Documentation/devicetree/bindings/media/allwinner,sun6i-a31-isp.yaml
index c0d7accc7bbe..a567e9db876d 100644
--- a/Documentation/devicetree/bindings/media/allwinner,sun6i-a31-isp.yaml
+++ b/Documentation/devicetree/bindings/media/allwinner,sun6i-a31-isp.yaml
@@ -13,6 +13,7 @@ properties:
   compatible:
     enum:
       - allwinner,sun6i-a31-isp
+      - allwinner,sun8i-a83t-isp
       - allwinner,sun8i-v3s-isp
 
   reg:
@@ -59,6 +60,7 @@ properties:
         compatible:
           contains:
             enum:
+              - allwinner,sun8i-a83t-isp
               - allwinner,sun8i-v3s-isp
     then:
       required:
-- 
2.43.0

